deployments/examples/ocis_full/mailserver.yml

```
@@ -0,0 +1,26 @@
+services:
+  ocis:
+    environment:
+      NOTIFICATIONS_SMTP_HOST: mailserver
+      NOTIFICATIONS_SMTP_PORT: 1025
+      NOTIFICATIONS_SMTP_SENDER: oCIS notifications <notifications@${OCIS_DOMAIN:-ocis.owncloud.test}>
+      NOTIFICATIONS_SMTP_USERNAME: notifications@${OCIS_DOMAIN:-ocis.owncloud.test}
+      NOTIFICATIONS_SMTP_INSECURE: "true"
+
+  mailserver:
+    image: axllent/mailpit:v1.22.3
+    networks:
+      - ocis-net
+    ports:
+      - "8025:8025" # Web UI
+      - "1025:1025" # SMTP
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.mailserver.entrypoints=https"
+      - "traefik.http.routers.mailserver.rule=Host(`${MAILPIT_DOMAIN:-mail.owncloud.test}`)"
```

rather than using specific service name use general words
```suggestion
      - "traefik.http.routers.mailserver.rule=Host(`${MAIL_DOMAIN:-mail.owncloud.test}`)"
```